qcore/timeseries.py

```diff
@@ -357,6 +357,16 @@ def vel2txt(self, station, prefix='./', title='', dt=None):
             seis2txt(c, dt, prefix, station, self.COMP_NAME[i],
                      start_sec=self.T_START, title=title)
 
+    def acc2txt(self, station, prefix='./', title='', dt=None):
+        """
+        Creates standard EMOD3D text files for the station (acc).
+        """
+        if dt is None:
+            dt = self.dt
+        for i, c in enumerate(self.acc(station, dt=dt).T):
```

Comment:
again, only differing in self.acc vs self.vel? parameter rather than copy and paste